server/resources/ats.json

```diff
@@ -55,10 +28,40 @@
             }
         }
     },
+    {
+        "id": 1,
+        "name": "JAWS",
+        "key": "jaws",
+        "vendorId": 1,
+        "defaultConfigurationInstructionsHTML": "Configure JAWS with default settings. For help, read &lt;a href=&quot;https://github.com/w3c/aria-at/wiki/Configuring-Screen-Readers-for-Testing&quot;&gt;Configuring Screen Readers for Testing&lt;/a&gt;.",
+        "assertionTokens": {
+            "screenReader": "JAWS",
+            "interactionMode": "PC cursor active",
+            "readingMode": "virtual cursor active",
+            "readingCursor": "virtual cursor"
+        },
+        "settings": {
+            "virtualCursor": {
+                "screenText": "virtual cursor active",
+                "instructions": [
+                    "Press &lt;kbd&gt;Alt&lt;/kbd&gt;+&lt;kbd&gt;Delete&lt;/kbd&gt; to determine which cursor is active.",
+                    "If the PC cursor is active, press &lt;kbd&gt;Escape&lt;/kbd&gt; to activate the virtual cursor."
+                ]
+            },
+            "pcCursor": {
+                "screenText": "PC cursor active",
+                "instructions": [
+                    "Press &lt;kbd&gt;Alt&lt;/kbd&gt;+&lt;kbd&gt;Delete&lt;/kbd&gt; to determine which cursor is active.",
+                    "If the virtual cursor is active, press &lt;kbd&gt;Insert&lt;/kbd&gt;+&lt;kbd&gt;z&lt;/kbd&gt; to disable the virtual cursor."
+                ]
+            }
+        }
+    },
     {
         "id": 3,
         "name": "VoiceOver for macOS",
         "key": "voiceover_macos",
+        "vendorId": 4,
```

Comment:
Why do we need to define the `vendorId` property for this patch? Would you mind if I moved JAWS back in order to make the diff clearer?

Reply:
Odd, just realized this happened. Looks to be a change included by #1208 that wasn't pushed. That can definitely be done separately

Reply:
Removed that change

Reply:
Awesome! I'll merge once CI passes